modules/channel-telegram/src/renderers/image.ts

```diff
@@ -0,0 +1,33 @@
+import * as sdk from 'botpress/sdk'
+import { ChannelRenderer } from 'common/channel'
+import { formatUrl } from 'common/url'
+import { TelegramContext } from '../backend/typings'
+
+export class TelegramImageRenderer implements ChannelRenderer<TelegramContext> {
+  get channel(): string {
+    return 'telegram'
+  }
+
+  get priority(): number {
+    return 0
+  }
+
+  get id() {
+    return TelegramImageRenderer.name
+  }
+
+  handles(context: TelegramContext): boolean {
+    return context.payload.image
```

Comment:
Shouldn't you return a boolean like so: `return !!context.payload.image`  instead of an any value?

Reply:
Good catch